lib/Instrumentation/AMDGPUHello.cpp

```diff
@@ -0,0 +1,70 @@
+#include "llvm/IR/IRBuilder.h"
+#include "llvm/IR/PassManager.h"
+#include "llvm/Pass.h"
+#include "llvm/Passes/PassBuilder.h"
+#include "llvm/Passes/PassPlugin.h"
+#include "llvm/Support/raw_ostream.h"
+#include <iostream>
+#include <vector>
+using namespace llvm;
+using namespace std;
+
+namespace {
+
+struct AMDGPUHello : public PassInfoMixin<AMDGPUHello> {
+  PreservedAnalyses run(Module &M, ModuleAnalysisManager &) {
+    bool ModifiedCodeGen = runOnModule(M);
```

Comment:
Please follow `cameCase` as the naming style. Applies to other variable names.

Reply:
I think I'm adhering to this throughout now (Variable names camel case and start with an upper case letter, function names camel case and start with a lower case letter). Let me know if I missed anything.